src/render/PanelHeaderProvider.js

```diff
@@ -89,13 +87,21 @@ export const PanelHeaderProvider = {
   getElementIcon: (element) => {
     const concreteType = getConcreteType(element);
 
+    const elementTemplates = getTemplatesService();
+
+    if (elementTemplates) {
+      const template = getTemplate(element, elementTemplates);
+
+      if (template && template.icon) {
+        return () => <img class="bio-properties-panel-header-template-icon" width="32" height="32" src={ template.icon.contents } />;
```

Comment:
Just for my understanding: We don't enforce Sizes in the renderer, but in the Properties panel we scale everything to 32/32. If we have a non-square Icon, this will be squashed/stretched it to the 1:1 aspect ratio.

Reply:
Yeah, just because the design in the properties panel header relies on a 32/32 icon, otherwise it would just look off. On the canvas, I assume we have more flexibility. 

We can always make it more relaxed here in the future though 👍